src/main/java/dk/kb/image/Facade.java

```diff
@@ -0,0 +1,174 @@
+package dk.kb.image;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+
+import javax.imageio.ImageIO;
+import javax.swing.plaf.ColorUIResource;
+
+import com.ctc.wstx.cfg.OutputConfigFlags;
+
+import java.awt.image.BufferedImage;
+import java.awt.Color;
+
+public class Facade {
+    public static byte[] getGreyscale(BufferedImage img) throws IOException{
```

Comment:
Please add JavaDoc for all `public` methods (also fine for `private`, but less important). For the greyscale this should include a general description (as always) and a specific description of what is returned (a JPEG file as bytes) as that is not clear from the method signature.

Reply:
JavaDoc added for all methods